src/utils/changeEventProducer.js

```diff
@@ -537,40 +341,13 @@ class ChangeEventProducer {
                 );
             }
         }
-        if (resourceType === 'Task') {
+        if (resourceType === 'Consent') {
             if (eventType === 'C') {
-                await this.onTaskCreateAsync({
-                    requestId, id: doc.id, resourceType, timestamp: currentDate, sourceType
-                });
-            } else if (doc.status === 'completed') {
-                await this.onTaskCompleteAsync({
-                    requestId, id: doc.id, resourceType, timestamp: currentDate, sourceType
-                });
-            } else if (doc.status === 'cancelled') {
-                await this.onTaskCanceledAsync({
-                    requestId, id: doc.id, resourceType, timestamp: currentDate, sourceType
-                });
-            } else {
-                await this.onTaskChangeAsync({
-                    requestId, id: doc.id, resourceType, timestamp: currentDate, sourceType
-                });
-            }
-        }
-        if (resourceType === 'Observation') {
```

Comment:
I don't believe we have any current use case around Observation events. And the cql-engine no longer produces Observation events anyway. So I think we can remove this for now. If we have a use case in the future for Observation events, we can emit them from a microservice involved with that work and not add the load to the FHIR server